ple/games/donkeykong/DonkeyKong.py

```diff
@@ -0,0 +1,103 @@
+__author__ = 'Erilyth'
+import pygame
+from Person import Person
+import random
+
+'''
+This class defines all the Donkey Kongs present in our game.
+Each donkey kong can only move on the top floor and cannot move vertically.
+'''
+
+
+class DonkeyKong(Person):
+    def __init__(self, raw_image, position):
+        super(DonkeyKong, self).__init__(raw_image, position)
+        self.__speed = 2
+        self.__direction = int(random.random() * 100) % 2
+        self.__cycles = 0
+        self.__stopDuration = 0
+
+    # Getters and Setters
+    def getSpeed(self):
+        return self.__speed
+
+    def setSpeed(self):
+        return self.__speed
+
+    def getStopDuration(self):
+        return self.__stopDuration
+
+    def setStopDuration(self, stopDuration):
+        self.__stopDuration = stopDuration
+
+    # Checks for collisions with walls in order to change direction when hit by a wall
+    def checkWall(self, colliderGroup):
+        if self.__direction == 0:
+            self.updateWH(self.image, "H", 20, 40, 40)  # Right collision with wall
+        if self.__direction == 1:
+            self.updateWH(self.image, "H", -20, 40, 40)  # Left collision with wall
+        Colliders = pygame.sprite.spritecollide(self, colliderGroup, False)
+        if self.__direction == 0:
+            self.updateWH(self.image, "H", -20, 40, 40)  # Right collision with wall
+        if self.__direction == 1:
+            self.updateWH(self.image, "H", 20, 40, 40)  # Left collision with wall
+        return Colliders
+
+    # This is used to animate the donkey kong
+    def continuousUpdate(self, GroupList, GroupList2):
+
+        # If the stop duration is 0 then kong is currently moving either left or right
+        if self.__stopDuration == 0:
+
+            # Currently moving right
+            if self.__direction == 0:
+                self.__cycles += 1
+                if self.__cycles % 24 < 6:
+                    self.updateWH(pygame.image.load('Assets/kong0.png'), "H", self.__speed, 40, 40)
```

Comment:
Performance wise: For all the `pygame.image.load` calls can you preload them in the `__init__` function and refer to the image by something like `self.IMAGE_KONG_0` so it doesnt need to be loaded each time the function is called.